src/NativeScript/inspector/InspectorPageAgent.mm

```diff
@@ -38,7 +40,18 @@
 }
 
 void InspectorPageAgent::reload(ErrorString&, const bool* in_ignoreCache, const String* in_scriptToEvaluateOnLoad) {
-    ASSERT_NOT_REACHED();
+    JSValue liveSyncCallback = m_globalObject->get(m_globalObject.globalExec(), Identifier::fromString(&m_globalObject.vm(), "__onLiveSync"));
+    if (!liveSyncCallback.isUndefined()) {
+        CallData callData;
+        CallType callType = getCallData(liveSyncCallback, callData);
```

Comment:
You should check whether `callType` is `CallTypeNone` and throw an exception through the inspector controller if so.